spec/imsc-hrm.html

```diff
@@ -244,14 +245,16 @@ <h2>Terms and Definitions</h2>
       Unless noted otherwise, this specification applies to an <a>IMSC Document Instance</a>.
     </p>
     <p>
-      Given a sequence of consecutive <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate
-      Synchronic Documents</dfn> generated using the <i>Intermediate Synchronic Document Construction</i> procedure
-      specified in [[!TTML2]] from one or more input <a>IMSC Document Instances</a>, the sequence of
-      <a>Intermediate Synchronic Documents</a> <i>conforms to the Hypothetical Render Model</i> SHALL be processed
-      without <a>error</a> by the HRM algorithm specified at <a href="#hypothetical-render-model-general"></a>.
+      A <a>IMSC Document Instance</a> <i>conforms to the Hypothetical Render Model</i> if the sequence of
+      <a>Intermediate Synchronic Documents</a> generated from it using the <i>Intermediate Synchronic Document
+      Construction</i> procedure specified in [[!TTML2]] is processed without <a>error</a> by the HRM algorithm
+      specified at <a href="#hypothetical-render-model-general"></a>.
     </p>
     <p class="note">Applying the Hypothetical Render Model to a <a>Document Instance</a> that is not an <a>IMSC Document
     Instance</a> yields results that might not reflect the complexity of the <a>Document Instance</a>.</p>
+    <p class="note">Conformance can be applied to a sequence of <a>Document Instances</a> by constructing a single
+    <a>Document Instance</a> whose underlying sequence of <a>Intermediate Synchronic Documents</a> is identical to that
+    resulting from the sequence of <a>Document Instances</a>.</p>
```

Comment:
I think we need to be a bit more careful with this note - and it's important that we have the note.

First, we need to introduce the idea that there may be some way to express a sequence of Document Instances, and some externally specified algorithm for resolving that sequence into a sequence of ISDs. Then we say that a Document Instance can be constructed that generates an equivalent sequence of ISDs, and that is what can be conformance checked. Equivalence need not go any further than what the HRM checks - "identical" is too high a bar. 

There are some potential cases where constructing a single Document Instance whose ISDs are identical is not possible. For example, two ISDs might contain elements with the same `xml:id` value, but not otherwise be the same, so the synthesised Document Instance would necessarily generate ISDs with different `xml:id`s on those elements. Presentationally the ISDs would be the same, and even more importantly, from the perspective of the HRM they would be indistinguishable.

Here's a suggestion:

```suggestion
    <p class="note">In applications where sequences of <a>Document Instances</a>
    can be resolved into a single sequence of <a>Intermediate Synchronic Documents</a>
    that do not overlap each other temporally,
    conformance can be determined based on a synthesised <a>Document Instance</a>
    that generates an equivalent sequence of <a>Intermediate Synchronic Documents</a>,
    where minimal equivalence is limited to the content and metrics that are used
    to identify <a>errors</a>.</p>
```